### PR TITLE
(Fix) SearchBar, Set Metadata bugfix, and refactoring

### PR DESCRIPTION
**Pull request by varasev**
- (Mandatory) Description

  This PR does the following things:
  * clears search bar (searchTerm) when network is switched
  * sets `window.ethereum.autoRefreshOnNetworkChange = true` since in new MetaMask versions the page reloading when network switching will be turned off by default
  * removes old excess files (`scripts` directory is used instead of `build_scripts`)
  * removes excess unused code and refactors some code
  * fixes the bug on `Set Metadata` page for `xDai` chain: https://forum.poa.network/t/anyblock-analytics-eth-events-as-an-xdai-validator/3138/11

- (Mandatory) What is it: (Fix), (Feature), or (Refactor)
(Fix, Refactor)

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -124,13 +123,13 @@ class AppMainRouter extends Component {
...
-        votingKey: account,
-        miningKey: miningKey
-      })
+  async onAccountChange(votingKey) {
+    const miningKey = await this.state.keysManager.miningKeyByVoting(votingKey)
+    this.setState({
+      votingKey,
+      miningKey
     })
+    console.log(`Accounts set:\nvotingKey = ${votingKey}\nminingKey = ${miningKey}`)
```
Comment (vbaranov) [inline src/index.js:132]: @varasev is this intentional having this log line on production?

**Review by vbaranov — APPROVED** [review]
Approved with some minor comments:

1. is about console.log
2. let's make popups consistent: `Metamask / Nifty Wallet` on both

Screenshot 2019-11-12 at 11 54 31
Screenshot 2019-11-12 at 11 54 08

File: src/index.js (hunk 1 above)
Comment (varasev) [reply src/index.js:132]: @vbaranov Yes, I added that to see the addresses in logs for the possible cases if validators report some possible bugs. That logs were in an old DApp's version, then were removed in one of the latest PRs (related to the drop-down network list), and then I returned those back in this PR.

**Comment by varasev** [comment]
> let's make popups consistent: `Metamask / Nifty Wallet` on both

OK, will do that.

**Pull request merged**